src/xGadgetron/cGadgetron/tests/CMakeLists.txt

```diff
@@ -67,18 +72,25 @@ target_link_libraries(MR_TESTS_CPP_AUXILIARY PUBLIC csirf cgadgetron)
 add_executable(MR_PROCESS_TESTDATA ${CMAKE_CURRENT_SOURCE_DIR}/mrtests_prep_testdata.cpp)
 add_dependencies(MR_PROCESS_TESTDATA GENERATE_SIMULATED_TESTDATA)
 target_link_libraries(MR_PROCESS_TESTDATA PUBLIC MR_TESTS_CPP_AUXILIARY csirf cgadgetron)
+set(MR_PROCESS_TESTDATA_EXE "${CMAKE_CURRENT_BINARY_DIR}/MR_PROCESS_TESTDATA")
+
 
 if(RERUN_ISMRMRD_SHEPP_LOGAN)
-        add_custom_target(PROCESS_SIMULATED_TESTDATA ALL
-                COMMAND MR_PROCESS_TESTDATA "${TESTDATA_TEMP_PREFIX}_1.h5" ${TESTDATA_OUTPUT_NAME_FULL}
-                COMMAND MR_PROCESS_TESTDATA "${TESTDATA_TEMP_PREFIX}_2.h5" ${TESTDATA_OUTPUT_NAME_GRAPPA}
-                COMMAND rm -f "${TESTDATA_TEMP_PREFIX}_1.h5"
-                COMMAND rm -f "${TESTDATA_TEMP_PREFIX}_2.h5"
-                WORKING_DIRECTORY ${TESTDATA_OUTPUT_PATH})
+  # this pre-processing step calls an executable MR_PROCESS_TESTDATA which fills a
+  # default geometry orientation into the MR acquisitions to suppress warnings.
+  add_custom_target(PROCESS_SIMULATED_TESTDATA ALL
+          COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$LD_LIBRARY_PATH ${MR_PROCESS_TESTDATA_EXE} "${TESTDATA_TEMP_PREFIX}_1.h5" ${TESTDATA_OUTPUT_NAME_FULL}
```

Comment:
This messing about with `LD_LIBRARY_PATH` fails on my Ubuntu VM. It actually runs
```
env LD_LIBRARY_PATH=/home/sirfuser/devel/build/SIRF/install/lib:D_LIBRARY_PATH 
```
I'll fix the use of the env variable, but obviously this would fail on OSX and Windows.

Reply:
@paskino same here